src/is-supported.ts

```diff
@@ -2,7 +2,7 @@ import { getMediaSource } from './utils/mediasource-helper';
 
 export function isSupported () {
```

Comment:
```suggestion
export function isSupported (): boolean {
```

I think it's always good to enforce the return type, because if catches things like someone adding

```ts
// window.someOtherApi = {};
return !!isTypeSupported && !!sourceBufferValidAPI && window.someOtherApi;
```

in the future